cornice_swagger/converters/parameters.py

```diff
@@ -24,10 +24,15 @@ def convert(self, schema_node, definition_handler):
 
         schema = definition_handler(schema_node)
 
-        if '$ref' in schema or schema['type'] == 'object':
+        if '$ref' in schema or schema.get('type') == 'object':
             converted['schema'] = schema
         else:
             converted['type'] = schema['type']
+
+        if schema.get('type') == 'array':
+            converted['items'] = {}
+            converted['items']['type'] = schema['items']['type']
```

Comment:
This is fine, but it could have been `converted['items'] = {'type': schema['items']['type']}`.